euphonic/validate.py

```diff
@@ -82,7 +82,8 @@ def _check_unit_conversion(obj: object, attr_name: str, attr_value: Any,
     if hasattr(obj, attr_name):
         if attr_name in unit_attrs:
             try:
-                _ = ureg(getattr(obj, attr_name)).to(attr_value)
+                _ = ureg(getattr(obj, attr_name)).to(attr_value,
```

Comment:
Does this need to return?

Reply:
By that, do you mean "why have the `_ = ...` or do you mean "this could be `ito` as we never use the result." ?

I think the `_ =` is just to clarify that this is being used for a side-effect. Do you find that distracting?

Performance seems fairly identical between `to` and `ito` here.

Reply:
Yeah, I mean why the `_ =`. To me it reads as a bit confusing as though the functionality might change if it doesn't return, but maybe that's my matlab-destroyed brain talking where that happens all the time.

Reply:
As far as I know that can't happen in Python! But to make a tenuous Matlab connection, this pattern is more often seen in Jupyter notebooks or interactive terminal to suppress the return value from being printed (like including a `;` in Matlab).